fabric-dimensions-v1/src/main/java/net/fabricmc/fabric/mixin/dimension/MixinChunkGenerator.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.mixin.dimension;
+
+import org.spongepowered.asm.mixin.Mixin;
+
+import net.minecraft.world.gen.chunk.ChunkGenerator;
+
+@Mixin(ChunkGenerator.class)
+public abstract class MixinChunkGenerator {
+	// Make this method to exist in dedicated servers
+	public abstract ChunkGenerator withSeed(long seed);
+
+	// Because that is not remapped
+	// Provide another one for production environment
+	public abstract ChunkGenerator method_27997(long seed);
```

Comment:
![concern](https://cdn.discordapp.com/emojis/707016186281590855.png?v=1)
This ain't the best solution... Notice loader may remap a mod to other namespaces such as spigotmap or searge names. Don't think these will work well in such situations.

Reply:
Do you have any solution

Reply:
Hmm, are you sure this method exists in all the subclasses of chunk generator? I need to look into the code in ide a bit

Reply:
This method does not exist in vanilla chunk generators in dedicated server. But as I tested it won't throw error indedicared server.

Reply:
Yes, this a far from ideal solution.